### ISSUE TITLE
How to install xdebug in the joseluisq/php-fpm:8.2

### ISSUE THREAD

**Issue by kristijorgji** [joseluisq/alpine-php-fpm#22]
I am using `joseluisq/php-fpm:8.2` and want to install on top of it xdebug for my local environment

My old development dockerfile fails because can't find xdebug pcl

```
FROM joseluisq/php-fpm:8.2 as base

FROM base as local

ARG DOMAIN
ARG NAME

USER root
RUN mkdir -p /etc/nginx/sites-enabled && ln -s /etc/nginx/sites-available/site /etc/nginx/sites-enabled/

# xdebug
RUN apk add php8-pecl-xdebug
# Xdebug config
COPY ./$NAME/xdebug_config.ini /etc/php8/conf.d/50_xdebug.ini

# utils
RUN apk add openssh \
    && apk add nano \
    && apk add git

#Enable ssh login
RUN apk add openrc && rc-update add sshd \
    && sed -i '/^#Port 22/s/^#//' /etc/ssh/sshd_config \
    && sed -i '/^#PasswordAuthentication/s/^#//' /etc/ssh/sshd_config \
    && sed -i '/^#PermitEmptyPasswords no/s/^#PermitEmptyPasswords no/PermitEmptyPasswords yes/' /etc/ssh/sshd_config \
    && sed -i '/^#PermitRootLogin prohibit-password/s/^#PermitRootLogin prohibit-password/PermitRootLogin yes/' /etc/ssh/sshd_config \
    && echo "root:Test" | chpasswd

# Permissions only for developingto allow docker-compose mount code
RUN echo -e 'user = 1000\ngroup = 1000' >> /etc/php8/php-fpm.d/www.conf
RUN chown -R root.root /var/www/html && \
  chown -R root.root /run && \
  chown -R root.root /var/lib/nginx && \
  chown -R root.root /var/log/nginx

COPY ./$NAME/nginx.conf /etc/nginx/nginx.conf

WORKDIR /var/www/html/app

COPY ./$NAME/entrypoint.sh /

ENTRYPOINT ["/entrypoint.sh"]

```

The error I get is

```
 > [local  2/10] RUN apk add php8-pecl-xdebug:
#6 0.205 fetch https://dl-cdn.alpinelinux.org/alpine/v3.17/main/aarch64/APKINDEX.tar.gz
#6 0.990 fetch https://dl-cdn.alpinelinux.org/alpine/v3.17/community/aarch64/APKINDEX.tar.gz
#6 1.751 ERROR: unable to select packages:
#6 1.768   php8-pecl-xdebug (no such package):
#6 1.768     required by: world[php8-pecl-xdebug]

```

**Comment by joseluisq** [comment]
`xdebug` extension is not included by default but it will be in future releases.
So in the meantime, you have to build the extension yourself.

```Dockerfile
RUN set -eux \
     && apk add --no-cache --virtual .build-deps autoconf gcc libc-dev linux-headers \
     && pecl install xdebug \
     && docker-php-ext-enable xdebug \
     && apk del .build-deps
```

After you can verify it

```sh
php -m | grep xdebug
# xdebug
```

**Comment by joseluisq** [comment]
Xdebug is now included in [v8.2.11](https://github.com/joseluisq/alpine-php-fpm/releases/tag/v8.2.11).
